github.md

```diff
@@ -1,13 +1,23 @@
 ---
 order: 1000
 ---
-# GitHub Workflows
+# GitHub 
+
+## Recommended repo settings
+
+In the admin settings for repos on GitHub, the recommended settings are: 
+* default branch should be "master" (Branches > Default branch)
+* disable force push to master. For everyone. Even admins. (Branches > Protected branches)
+    * check: `protect master branch`, `require status checks`, `require that branches are up to date`, `include administrators`, `require CI to pass` if available
```

Comment:
I removed status checks though the setting for making sure the branch is up-to-date is under that checkbox. If you're cool with retaining that check, i'll add that back in with more detail.